python/cugraph/cugraph/centrality/katz_centrality.py

```diff
@@ -11,11 +11,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from pylibcugraph.experimental import (ResourceHandle,
-                                       GraphProperties,
-                                       SGGraph,
-                                       katz_centrality as pylibcugraph_katz
-                                       )
+from pylibcugraph import (ResourceHandle,
+                          GraphProperties,
+                          SGGraph,
+                          )
+from pylibcugraph.experimental import katz_centrality as pylibcugraph_katz
```

Comment:
Can you promote this to stable, since we're using it here in a stable API?  If not, users using `cugraph.katz_centrality()` will see the `experimental` warning.  Since we never released with `pylibcugraph.experimental.katz`, there's no need to use the `promoted_experimental_warning_wrapper()` and you should be able to promote it in this PR.

Reply:
Yes, following our discussion on slack I removed promoted_experimental_warning_wrapper from any of the eigenvector_centrality wrappers, and addressed all of the comments from yesterday.